acceptance-tests/src/test/java/io/blueocean/ath/offline/multibranch/CommitMessagesTest.java

```diff
@@ -41,12 +47,19 @@
     @Inject
     MultiBranchPipelineFactory mbpFactory;
 
+    @Inject
+    ClassicJobApi jobApi;
+
+    @After
+    public void tearDown() throws IOException {
```

Comment:
Do we really want this? It might make debugging harder if you can't mess with jenkins via VNC. The whole thing gets splatted at the end, on account of it being a container that lives only until run.sh ends

Reply:
Yea, sadly, some of the tests, like favs, are actually clicking on items, and based on some of the github creation tests retrying, sometimes there's too many items and the fav items show up on the next page. So I got a little agressive on cleaning up tests.

I agree, it does make things a bit harder, but you can always comment it out or something right?

Reply:
Yeah makes sense. Just checking there's a reason is all :)